Orange/evaluation/scoring.py

```diff
@@ -132,6 +132,39 @@ def compute_score(self, results, target=None):
                     dtype=np.float64, count=len(results.predicted))
 
 
+class LogLoss(Score):
+    """Log loss, aka logistic loss or cross-entropy loss.
```

Comment:
Docstring for sklearn wrappers should start with the following disclaimer:

A wrapper for sklearn.linear_model.logistic.LogisticRegression. The following is the documentation from scikit-learn.